phone.go

```diff
@@ -67,7 +67,7 @@ func (p Phone) tollfreephonenumber() string {
 		}
 		out += v
 	}
-	return fmt.Sprintf("(%s) %s", boxDigitsStart[rand.Intn(1)], out)
+	return fmt.Sprintf("(%s) %s", boxDigitsStart[rand.Intn(len(boxDigitsStart))], out)
```

Comment:
`rand.Intn(1)` always returns 0.
So, `boxDigitsStart[rand.Intn(1)]` always is "777".

This fix is to correctly return random values.